sources/packages/forms/src/form-definitions/sfaa2021-22.json

```diff
@@ -16589,7 +16589,7 @@
               "inputFormat": "plain",
               "validate": {
                 "required": true,
-                "custom": "",
+                "custom": "valid = (input >= 0 && input < 100000000) ? true : 'The number you have entered is not an acceptable value.'",
```

Comment:
Is this an acceptable format? I believe it must be 100.0
![image](https://github.com/bcgov/SIMS/assets/54600590/24d73f53-6d33-4724-9c9f-45e2672b9e86)

Reply:
If it is something configurable we can adjust, otherwise, if it is the out-of-box behavior of the component I would recommend not investing time unless it would cause some harm. For instance, if the value being saved is different than "100" in the above case.

Reply:
It is the default behavior in form.io even if decimal input is disabled for the input field. The data will not display "." when it is entered. For example, 100. will be saved as 100 in the database.

Reply:
@dheepak-aot @andrewsignori-aot I found that the prepared payload does not have the decimal and hence only **100** goes through to the database and not **100.** as @lewischen-aot suggested.